configure_workers_and_start.py

```diff
@@ -2006,10 +2006,9 @@ def main(args: List[str], environ: MutableMapping[str, str]) -> None:
         getenv_bool("SYNAPSE_ENABLE_POSTGRES_METRIC_EXPORT", False)
         and "POSTGRES_PASSWORD" in environ
     )
-    prometheus_storage_retention_time = (
-        os.getenv("PROMETHEUS_STORAGE_RETENTION_TIME", "1y")
-        and enable_prometheus is True
-    )
+    if enable_prometheus:
```

Comment:
Actually, could probably just drop the `enable_prometheus` part altogether I think. If memory serves, it's ultimate purpose was to flip the switch to `ON` inside the supervisord configuration. If it's disabled, then loading a default value is handled from the `getenv()` function(look for the "1y") and it would just be ignored in the grand scheme. 

To be fair it has been a while, but please double check that makes sense to you

Reply:
Ah okay yes that is a good idea. This is just setting the variable and no other logic is applied.